ui/src/helpers/utils.ts

```diff
@@ -0,0 +1,6 @@
+export const simplifyDate = (value: string | Date) => {
+	const date = new Date(value)
+	return `${
+		date.getMonth() + 1
+	}/${date.getDate()} ${date.getUTCHours()}:${date.getMinutes()}:${date.getSeconds()}`
```

Comment:
From our timezone, if the time was close to midnight, the hours would push into the next day, but the date wouldn't change.

You might consider using `date-fns` to help, we use it in other projects. https://date-fns.org/docs/Getting-Started
It would be something like: `format(new Date(value), 'd/M HH:mm:ss')`